config.m4

```diff
@@ -502,7 +477,7 @@ if test "$MONGODB" != "no"; then
       AC_OUTPUT($srcdir/src/libmongoc/src/zlib-1.2.11/zconf.h)
     fi
   fi
-  if test "$PHP_LIBBSON" == "no"; then
+  if test "$PHP_LIBBSON" = "no"; then
```

Comment:
Was this always broken like this before? Or do both = and == work (on bash)?

Reply:
Bash supports both `==` and `=`, but `=` is POSIX compliant. You'll note from the new shell scripts that I'm aiming to keep things portable with `#!/bin/sh` and POSIX-friendly syntax.